src/index.ts

```diff
@@ -130,15 +134,21 @@ function requestToFetchOptions(reqOpts: Options) {
     uri = uri + '?' + params;
   }
 
+  const isHttp = /^http:\/\//.test(uri);
```

Comment:
This is a total nit, and calls out how lame of a programmer I am.  Instead of using a regex - could we just say:

```js
const isHttp = uri.startsWith('http://');
```

Reply:
I think you're right, I still use `contains` and regexes more than I think I need to, since I haven't done as well updating my brain to reflect JavaScript's API improvements.